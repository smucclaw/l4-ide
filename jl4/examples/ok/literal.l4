§ Literals

DECIDE primes IS
    2 FOLLOWED BY
    3 FOLLOWED BY
    5 FOLLOWED BY
    7 FOLLOWED BY
    11 FOLLOWED BY
    EMPTY

DECIDE `another definition of primes` IS
    LIST
      2, 3, 5
      7, 11

DECIDE countries IS
    "Germany" FOLLOWED BY
    "Österreich" FOLLOWED BY
    "新加坡共和国" FOLLOWED BY -- Unicode test
    "Some \"country\"" FOLLOWED BY -- escapes
    EMPTY

