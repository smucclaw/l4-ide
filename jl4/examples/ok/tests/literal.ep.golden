ยง Literals

DECIDE primes IS
    2 FOLLOWED BY
    3 FOLLOWED BY
    5 FOLLOWED BY
    7 FOLLOWED BY
    11 FOLLOWED BY
    EMPTY

DECIDE `another definition of primes` IS
    LIST
      2, 3, 5
      7, 11

DECIDE countries IS
    "Germany" FOLLOWED BY
    "\214sterreich" FOLLOWED BY
    "\26032\21152\22369\20849\21644\22269" FOLLOWED BY -- Unicode test
    "Some \"country\"" FOLLOWED BY -- escapes
    EMPTY

